src/utils/useFieldState.ts

```diff
@@ -0,0 +1,50 @@
+import React, {useCallback, useState} from "react";
+import {SkjemaelementFeil} from "nav-frontend-skjema/lib/skjemaelement-feilmelding";
```

Comment:
tror det er feil autocomplete import fra intelliJ?

Reply:
IntelliJ viser ikke alternativer å velge fra. Hva tenker du er mer riktig?

Reply:
Test om `import { SkjemaelementFeil } from 'nav-frontend-skjema'` fungerer. 
Men det er en TS-type som importeres her, så kan hende vi må bruke full path for at TS-compileren skal forstå hva som foregår.

Reply:
gir feilmelding " .. nav-frontend-skjema/lib"' has no exported member 'SkjemaelementFeil'.  TS2305"